physics/equipotential_test.cpp

```diff
@@ -407,17 +422,23 @@ TEST_F(EquipotentialTest, BodyCentredBodyDirection_GlobalOptimization) {
   };
   const MultiLevelSingleLinkage<SpecificEnergy, Position<World>, 2>::Box box = {
       .centre = World::origin,
-      .vertices = {Displacement<World>({1'000'000 * Kilo(Metre),
+      .vertices = {Displacement<World>({3 * Metre,
                                         0 * Metre,
                                         0 * Metre}),
                    Displacement<World>({0 * Metre,
-                                        1'000'000 * Kilo(Metre),
+                                        3 * Metre,
                                         0 * Metre})}};
 
   MultiLevelSingleLinkage<SpecificEnergy, Position<World>, 2> optimizer(
       box, potential, acceleration);
   Equipotential<Barycentric, World> const equipotential(
-      equipotential_parameters_, &reference_frame);
+      {EmbeddedExplicitRungeKuttaIntegrator<
+           DormandPrince1986RK547FC,
+           Equipotential<Barycentric, World>::ODE>(),
+       /*max_steps=*/1000,
+       /*length_integration_tolerance=*/1 * Nano(Metre)},
+      &reference_frame,
+      /*characteristic_length=*/1 * Nano(Metre));
```

Comment:
Could we put that length in a constant and reference it wherever we have lengths?